modeler documentation review (#2427)
camunda/camunda-platform-docs: docs/components/modeler/about-modeler.md

<span class="badge badge--cloud">Camunda Platform 8 only</span>

Any executable process needs a **[Business Process Model and Notation (BPMN)](./bpmn/bpmn.md) diagram** designed and configured beforehand.

A BPMN diagram is used to visually outline the structure and flow of a process. As a result, the process can be more easily understood by various stakeholders.

In tandem, different events and implementation details (such as the conditions within a gateway or the specifications of a service task) must be configured in the model so the workflow engine understands what must be executed once the process reaches a certain task.

> Camunda offers two tools to design and implement your diagrams:

Reviewer: ```suggestion
Camunda offers two tools to design and implement your diagrams: [Web Modeler](./web-modeler/launch-cloud-modeler.md) offers a seemless integration in into Camunda Platform 8 SaaS and self-managed installations. [Desktop Modeler](./desktop-modeler/index.md) is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.
```

Author: See https://github.com/camunda/camunda-platform-docs/pull/2427#discussion_r1294823491 👍

> - [Web Modeler](./web-modeler/launch-cloud-modeler.md)

Reviewer: ```suggestion
```

Author: See https://github.com/camunda/camunda-platform-docs/pull/2427#discussion_r1294823491 👍

- [Desktop Modeler](./desktop-modeler/index.md)

> Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.

Reviewer: We could probably re-write this whole section:

```markdown
Camunda offers two tools to design and implement your diagrams: [Web Modeler](./web-modeler/launch-cloud-modeler.md) offers a seemless integration in into Camunda Platform 8 SaaS and self-managed installations. [Desktop Modeler](./desktop-modeler/index.md) is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.
```

> Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.

Author: Thank you @nikku! I adjusted with some active voice:

> Camunda offers two tools to design and implement your diagrams:

- [Web Modeler](./web-modeler/launch-cloud-modeler.md): Integrate seamlessly with Camunda Platform 8 SaaS and Self-Managed installations alongside [Console](../console/introduction-to-console.md).
- [Desktop Modeler](./desktop-modeler/index.md): Design, view, and edit models using this desktop application. Install and use Desktop Modeler locally, all while integrating your local development environment.

> Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.

Author: See https://github.com/camunda/camunda-platform-docs/pull/2427#discussion_r1294823491 👍

> Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Platform 8 to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally to also design, view, and edit models, all while integrating your local development environment.

Reviewer: Sounds great.

:::note

> Interested in editing the underlying XML of BPMN and [Decision Model and Notation (DMN)](./dmn/dmn.md) diagrams directly? Be sure to utilize Desktop Modeler, as it features an XML editor tab. The XML editor comes with its own history (undo or redo), search and replace functionality, and XML syntax highlighting.

Reviewer: Why do we link only one of both tools here? Let's link both for consistency.

Author: I usually link on first reference, so BPMN is linked on line 10. Does this feel awkward to you?

Reviewer: I often dive into individual sections of a documentation page (and don't assume this is read top -> bottom). Hence I'd consistently link both here. 

Only linking first time is what I know works from documents that are read strictly top to bottom.

Author: Adjusted in latest commit 👍

:::

In this guide, we'll demonstrate modeling BPMN diagrams using both Web Modeler and Desktop Modeler.

## Next steps

- [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to model an automated process in this tutorial using Web Modeler with Camunda Platform 8 SaaS.

> - [Camunda Forms](/guides/utilizing-forms.md) - Allows you to design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.

Reviewer: To substantially improve this section we could ensure each bullet point encourages the user to dive into it.

#### Good

> Learn how to ...

#### Not so good.

> X is a feature that does Y.

----

```suggestion
- [Camunda Forms](/guides/utilizing-forms.md) - Design and configure user task forms to implement human task orchestration use-cases.
```

- [DMN](./dmn/dmn.md) - In DMN, decisions can be modeled and executed using the same language. Business analysts can model the rules that lead to a decision in easy to read tables, and those tables can be executed directly by a decision engine (like Camunda).

> - [Out-of-the-box Connectors](/guides/configuring-out-of-the-box-connector.md) - Out-of-the-box Connectors provide prebuilt connectivity with a number of outside systems. Connectors are modular by nature, reusable, and accelerate automation across those systems.

Reviewer: Proposal:

* Remove this and/or introduce a section called "discover/use connectors" => It is powerful, and we want people to know them.

> - [Creating Connector templates](/components/connectors/manage-connector-templates.md) - Connector templates allow you to create and build connectivity with the outside systems you choose and want to automate across in Modeler.

Author: Does this feel too repetitive?

- [Out-of-the-box Connectors](/guides/configuring-out-of-the-box-connector.md) - Out-of-the-box Connectors provide prebuilt connectivity with a number of outside systems. Connectors are modular by nature, reusable, and accelerate automation across those systems.
- [Custom connectors](/components/connectors/manage-connector-templates.md) - Learn how to build custom Connectors alongside external systems based on our out-of-the-box Connectors.
